packages/gatsby-remark-autolink-headers/src/gatsby-ssr.js

```diff
@@ -13,10 +13,16 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   )
 
   const styles = `
-    .${className} {
-      float: left;
+    .${className}.before {
+      position: absolute;
+      top: 0;
+      left: 0;
+      transform: translateX(-100%);
       padding-right: 4px;
-      margin-left: -20px;
+    }
+    .${className}.after {
+      display: inline-block;
+      padding-left: 4px;
```

Comment:
This change also changes the css of the `left` case. Was that necessary?

Reply:
@fk can you take a quick look at this?

It looks fine to me but I'm not sure if this will be a problem for anything else in the gatsby css world. Cheers

Reply:
LGTM, but haven't built locally.